privacyidea/static/contrib/css/bootstrap-theme.css

```diff
@@ -309,6 +309,32 @@ fieldset[disabled] .btn-danger.active {
   background-color: #c12e2a;
   background-image: none;
 }
+.btn-transparent{
```

Comment:
The `contrib` folder contains the data from external packages (bootstrap, angular and plugins). This will be overwritten in the next update of the packages.
Could You move these changes to the `static/css` folder? Or are these official changes of a newer bootstrap version?